advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md

```diff
@@ -18,6 +18,7 @@ It's important to understand that while the WCAG can be incredibly helpful for b
 ### The Four Principles
 
 The WCAG are organized around four, core principles that should be kept in mind when implementing any sort of accessibility feature:
```

Comment:
Could we fit the acronym in more naturally?

Maybe with something like this:
```suggestion
The WCAG are organized around four, core principles (POUR) that should be kept in mind when implementing any sort of accessibility feature:
```

Reply:
Great idea. How can I change my pull request 😅.Kinda new to GitHub myself

Reply:
No worries, you can edit the file directly in the file [changed tab](https://github.com/TheOdinProject/curriculum/pull/25368/files)

<img width="1225" alt="Screenshot 2023-04-19 at 11 10 13" src="https://user-images.githubusercontent.com/7963776/233043446-27b8d4c0-12d1-4ecc-996b-a494e67cb153.png">